builtin/game/falling.lua

```diff
@@ -146,13 +146,28 @@ end
 
 local function drop_attached_node(p)
 	local n = core.get_node(p)
+	local node_meta = core.get_meta(p)
+	local meta_fields = node_meta and node_meta:to_table().fields end
 	core.remove_node(p)
 	for _, item in pairs(core.get_node_drops(n, "")) do
 		local pos = {
 			x = p.x + math.random()/2 - 0.25,
 			y = p.y + math.random()/2 - 0.25,
 			z = p.z + math.random()/2 - 0.25,
 		}
+		-- If it drops itself, preserve its metadata.
+		if item == n and meta_fields then
+			local stack = ItemStack({name=n, count=1})
+			local stack_meta = stack and stack:get_meta() end
+			if stack_meta then
```

Comment:
just checked the code, get_meta() on an itemstack can't fail

Reply:
Nice to know.  But the get_meta() will never execute if ItemStack() returns nil.

Reply:
Itemstack will never return nil

Reply:
Also nice to know :-)

What I can't seem to find is a button in github asking for a fresh review from somebody.
Do I just reply to their comments to trigger a new review request?